### PR TITLE
[MOS-1014] Screen remains black after turning off the phone with dark…

### PR DESCRIPTION
**Pull request by lukasmas**
… mode enabled

Regression fixed by adjusting wipeout function.

**Your checklist for this pull request**

Make sure that this PR:
- [x] Complies with our guidelines for contributions
- [ ] Has unit tests if possible
- [ ] Has documentation updated
- [ ] Has changelog entry added

### CODE REVIEW

File: module-bsp/board/rt1051/bsp/eink/ED028TC1.cpp [hunk 1]
```diff
@@ -925,6 +925,7 @@ EinkStatus_e EinkFillScreenWithColor(EinkDisplayColorFilling_e colorFill)
     buf[8] = EINK_DISPLAY_RES_Y & 0x00FF;
 
     if (BSP_EinkWriteData(buf, 9, SPI_AUTOMATIC_CS) != 0) {
+        BSP_EinkWriteCS(BSP_Eink_CS_Set);
```
Comment (Lefucjusz) [inline module-bsp/board/rt1051/bsp/eink/ED028TC1.cpp:928]: Is this change really required? AFAIR calling `EinkWriteData` with `SPI_AUTOMATIC_CS` should already leave `~CS` line in high state after the transmission has completed

**Review by Lefucjusz — APPROVED** [review]

**Review by dawidwojtas-mudita — APPROVED** [review]

**Review by Maciej-Mudita — APPROVED** [review]

File: module-bsp/board/rt1051/bsp/eink/ED028TC1.cpp (hunk 1 above)
Comment (lukasmas) [reply module-bsp/board/rt1051/bsp/eink/ED028TC1.cpp:928]: You are right. It is not needed, I will remove it
